lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -120,15 +120,11 @@ export default function ErrorDisplay({ children, description = '', error }) {
 
   return (
     <Scrollbox classes="LMS-Scrollbox">
-      <div className="hyp-u-vertical-spacing pt-4 space-y-4">
-        {message && (
-          <p className="m-0" data-testid="error-message">
-            {toSentence(message)}
-          </p>
-        )}
+      <div className="pt-4 space-y-4">
+        {message && <p data-testid="error-message">{toSentence(message)}</p>}
 
         {children}
-        <p data-testid="error-links" className="m-0">
+        <p data-testid="error-links">
```

Comment:
No longer need the combination of margin suppression or some rules in `hyp-u-vertical-spacing` to make this lay out correctly. Phew.